client/src/components/SendgridOps.js

```diff
@@ -343,17 +336,17 @@ export function deleteRefreshr() {
     .catch(err => console.log(err))
 }
 
-export function scheduleRefreshr() {
-  const scheduleObj = {
-    "send_at": 1551103200 // Feb 25, 2019 8AM
-  }
+export function scheduleRefreshr(scheduleObj, campaign_id) {
   const url = `https://api.sendgrid.com/v3/campaigns/${campaign_id}/schedules`
```

Comment:
I feel like this is where the magic happens.

Reply:
That's it! Haha